ha/core/event_analyzer/parser/parser.py

```diff
@@ -159,15 +159,102 @@ class ClusterResourceParser(Parser):
     Subscriber for event analyzer to pass msg.
     """
 
-    def __init__(self):
+    def __init__(self, conf_store=None):
         """
         Init method.
         """
         super(ClusterResourceParser, self).__init__()
         ConfigManager.init("event_analyzer")
+        self._conf_store = conf_store
+        self.cluster_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}cluster_id")
+        self.site_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}site_id")
+        self.rack_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}rack_id")
+        self.predefined_config = {"cluster_id": self.cluster_id, "site_id": self.site_id, "rack_id": self.rack_id}
+        self.system_health_node_key_template = "cortx/ha/system/cluster/$cluster_id/site/$site_id/rack/$rack_id/node/$node_id/health"
         Log.info("ClusterResource Parser is initialized ...")
 
-    def parse_event(self, msg: str) -> HealthEvent:
+    def prepare_key(self, res_alert: dict) -> str:
```

Comment:
I think all system health related functionality should be in system health module

Reply:
Right. DOne.